proxy/src/config.rs

```diff
@@ -100,6 +103,7 @@ pub enum UrlError {
 // Environment variables to look at when loading the configuration
 const ENV_EVENT_BUFFER_CAPACITY: &str = "CONDUIT_PROXY_EVENT_BUFFER_CAPACITY";
 const ENV_METRICS_FLUSH_INTERVAL_SECS: &str = "CONDUIT_PROXY_METRICS_FLUSH_INTERVAL_SECS";
+const ENV_REPORT_TIMEOUT_SECS: &str = "CONDUIT_PROXY_REPORT_TIMEOUT_SECS";
```

Comment:
If we have timeouts expressed as seconds then we can't write tests for this that time out in less than a second without the tests skipping over the `Config` structure's parsing of the settings strings, which reduces the test coverage of the settings strings parsing.

This is also inconsistent with other timeouts like `ENV_PRIVATE_CONNECT_TIMEOUT` and `ENV_PUBLIC_CONNECT_TIMEOUT` which are set in milliseconds already.